apps/examples/testcase/le_tc/kernel/tc_libc_math.c

```diff
@@ -2026,6 +2026,32 @@ static void tc_libc_math_ldexpl(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+ * @fn                   :tc_libc_math_log
+ * @brief                :Returns the binary logarithm of x
```

Comment:
I think its not binary logarithm. Its natural logarithm(log to base 'e')

Reply:
modified comments accordingly.